src/h5web/vis-packs/core/shared/VisMesh.tsx

```diff
@@ -9,22 +9,10 @@ interface Props extends MeshProps {
 function VisMesh(props: Props) {
   const { children, ...meshProps } = props;
 
-  const {
-    abscissaConfig,
-    ordinateConfig,
-    abscissaScale,
-    ordinateScale,
-    visSize,
-  } = useAxisSystemContext();
-
-  const [minAbscissa, maxAbscissa] = abscissaConfig.visDomain;
-  const [minOrdinate, maxOrdinate] = ordinateConfig.visDomain;
-
-  const x = abscissaScale(minAbscissa + (maxAbscissa - minAbscissa) / 2);
-  const y = ordinateScale(minOrdinate + (maxOrdinate - minOrdinate) / 2);
```

Comment:
Not only this calculation to find the center of the vis is wrong for non-linear scales, but it is completely indeed as since https://github.com/silx-kit/h5web/pull/705, the center of the vis correspond to the center of the canvas by construction.

Therefore, we can remove the calculation and simply set the `position` to the center of the canvas `[0, 0, 0]`